packages/loopring_v3/contracts/core/impl/ExchangeV3.sol

```diff
@@ -613,6 +619,23 @@ contract ExchangeV3 is IExchangeV3
         emit TransactionApproved(owner, transactionHash);
     }
 
+    function approveTransactions(
+        address[] calldata owners,
+        bytes32[] calldata transactionHashes
+        )
+        external
+        override
+        nonReentrant
+        onlyFromUserOrAgent(owner)
```

Comment:
remove (unfortunately a bit dangerous that the owner of the contract is also just `owner`, which we often use as the name of our function inputs).